piptools/utils.py

```diff
@@ -371,3 +371,21 @@ def get_compile_command(click_ctx):
                 )
 
     return " ".join(["pip-compile"] + sorted(left_args) + sorted(right_args))
+
+
+def create_install_command():
+    """
+    Return an instance of InstallCommand.
+    """
+    if PIP_VERSION < (10,):
+        from pip.commands.install import InstallCommand
+
+        return InstallCommand()
+    elif PIP_VERSION < (19, 3):
+        from pip._internal.commands.install import InstallCommand
+
+        return InstallCommand()
+
+    from pip._internal import create_command
+
+    return create_command("install")
```

Comment:
You could use `do_import` in this method then - if it results in less code.
With more lazily created commands it makes sense to abstract this pattern anyway.
But can be done later also I guess.

Reply:
Should i revert two latest commits then?

Reply:
@blueyed kindly ping

Reply:
Addressed in 4e7404f1216f763e7b325d30a02887d6fbb267cf